docs/layer.md

```diff
@@ -25,53 +23,56 @@ which helps avoid id collisions in this case.
 
 E.g. assuming a composite GeoJsonLayer layer that renders two sublayers,
 choropleths and lines, with those ids:
-```
```

Comment:
If you indent your code with at least 4 spaces it will be code-formatted, this way you don't have to add extra markup to do it, and it removes the extra left-padding. We could keep the triple backtick, but would need to remove the extra spaces on each line.